bot/constants.py

```diff
@@ -49,6 +49,9 @@
 GREEN_CHEVRON = "<:greenchevron:418104310329769993>"
 RED_CHEVRON = "<:redchevron:418112778184818698>"
 WHITE_CHEVRON = "<:whitechevron:418110396973711363>"
+PAID_ICON = "<:paid_icon:432872213088305153>"
+FREEWARE_ICON = "<:freeware_icon:432872213088305153>"
+FREE_ICON = "<:free_icon:432872213088305153>"
```

Comment:
```python
PAID_ICON = "<:paid_icon:432849192562458644>"`
FREEWARE_ICON = "<:freeware_icon:432848334143750144>"`
FREE_ICON = "<:free_icon:432864242715197441>"
```
I know you probably changed them for testing, but they gotta be right if we want to merge